packages/adblocker/src/resources.ts

```diff
@@ -333,6 +333,24 @@ export default class Resources {
     return script;
   }
 
+  private getResourceAsScriptlet(name: string): Scriptlet | undefined {
```

Comment:
if this is private and the the only user need only the body then return a body only